src/azure-cli/azure/cli/command_modules/acs/managed_cluster_decorator.py

```diff
@@ -2359,6 +2359,52 @@ def get_network_dataplane(self) -> Union[str, None]:
         :return: str or None
         """
         return self.raw_param.get("network_dataplane")
+    
+    def get_acns(self) -> Union[bool, None]:
+        """Get the enablement of acns
+        
+        :return: bool or None"""
+        enable_acns = self.raw_param.get("enable_acns")
+        disable_acns = self.raw_param.get("disable_acns")
+        if enable_acns and disable_acns:
+            raise MutuallyExclusiveArgumentError(
+                "Cannot specify --enable-acns and "
+                "--disable-acns at the same time."
+            )
```

Comment:
add a `MutuallyExclusiveArgumentError` for when` --enable-acns`, `--disable-acns-observability` and `--disable-acns-security` are ran at the same time